pandas/tests/indexes/datetimes/test_timezones.py

```diff
@@ -731,6 +731,18 @@ def test_time_accessor(self, dtype):
 
         tm.assert_numpy_array_equal(result, expected)
 
+    @pytest.mark.parametrize('tz', [
+        dateutil.tz.gettz('US/Eastern'), dateutil.tz.gettz('US/Pacific'),
+        dateutil.tz.gettz('UTC')])
+    def test_timetz_accessor(self, tz):
```

Comment:
Instead could you use the `tz_naive_fixture`?

https://github.com/pandas-dev/pandas/blob/26b3e7de56fdebaf1842b01d747a1a30a126c54a/pandas/conftest.py#L290